articles/Silo_Finance.md

```diff
@@ -0,0 +1,448 @@
+
+When depositing tokens, Silo provides an innovative feature that allows lenders to choose whether their deposits can be borrowed by others, or whether they want to “protect” their deposits, which disables others from borrowing said assets. However, in return for protecting the token, the lender forgoes earning an interest rate. _sTokens_ represent borrowable assets, and _spTokens_ represent protected deposits. It’s also possible to do both for the same asset (e.g. 40% of the token is borrowable and 60% is protected). _spTokens_ can still be used as collateral to borrow other tokens.
+
+For bridge assets (ETH or XAI), the naming convention is slightly different, i.e._ sBridge-ABC_, whereby ABC represents the name of the silo that ETH or XAI is deposited into. For instance, when depositing the bridge asset ETH into the LINK silo, the lender in return receives _sETH-LINK_ or _spETH-LINK_.
+
+
+## Debt Tokens (dTokens)
+
+Conversely, [dTokens](https://silopedia.silo.finance/borrow/debt-tokens-dtokens) are ERC-20 tokens representing a borrowed asset. They are minted when an asset is borrowed and burned when the loan is paid back. Unlike _sTokens_, _dTokens_ cannot be transferred out of a wallet. For bridge assets, the borrowed token is represented again as _dBridge-ABC_ (e.g. dETH-UNI).
+
+
+## Conclusion
+
+In summary, Silo introduces a few innovations and a new money market primitive aiming to make lending markets less risky, while simultaneously improving market access for long-tail assets. The ability to protect tokens from borrowers can be a very interesting feature for other DAOs to deploy their tokens and borrow tokens against them.
+
+The risk reduction, however, has one trade-off with the current design: Borrowing at Silo can result in up to six transactions. First, &lt;allow deposit> and &lt;deposit> of the collateral, then &lt;borrow> the bridge asset, then again &lt;allow deposit> and &lt;deposit> the bridge asset, before finally the targeted asset can be &lt;borrowed>. This can be quite costly on the Ethereum mainnet and might be a blocker for smaller accounts to use Silo as intended. The team is working on new features to improve this, but it will take some time.
+
+
+# The XAI Stablecoin
+
+The Silo team aims to issue a new over-collateralized stablecoin with a soft peg to the US Dollar called XAI. The main use case for XAI is to serve as a second bridge token alongside ETH. Thus, enabling users to borrow XAI to bridge from token A to token B.
+
+SiloDAO is the only entity that controls XAI. The DAO can choose to mint unlimited XAI and deposit it into any number of silos via executive proposals. The DAO can also burn XAI that is extended to any silos via governance proposals. Hence, when minting XAI into a silo, the SiloDAO effectively determines XAI’s backing, which can also be reversed by burning XAI from a silo.
```

Comment:
`any number of silos via executive proposal` Let's mention they plan to limit initially only to ETH bridge silo and USDC silo, as this limits the risk surface of its backing (which will need to be continuously evaluated by the DAO as new collateral types are added).

Reply:
added